public/src/components/channelManagement/variantEditorSeparateArticleCountEditor.tsx

```diff
@@ -18,6 +24,20 @@ const VariantEditorSeparateArticleCountEditor: React.FC<VariantEditorSeparateArt
     updateSeparateArticleCount(Boolean(separateArticleCount) ? undefined : { type: 'above' });
```

Comment:
the way this worked before means that the copy will be lost if "Above" is set to false, which I guess is fine?